### ISSUE TITLE
Use Case 8 : Divide Tasks Between Teams

### ISSUE THREAD

**Issue by ktageja** [ktageja/DataSense-PRJ666-team-5#10]
## User Story
As a project manager or team lead, I want to divide tasks between development teams so that each team focuses on specific parts of the project.
## Description
This use case describes how the project manager or team leads assign tasks to specific development teams (e.g., frontend, backend, hardware) based on the requirements of the sprint.

Efficient task distribution is critical to ensure that all aspects of the project progress in parallel, allowing teams to focus on their areas of expertise.

## Acceptance Criteria

- [ ] Tasks are clearly assigned to relevant teams (frontend, backend, hardware).
- [ ]  Task assignments are communicated via GitHub Projects or another task management tool.
- [ ]  Teams are aware of their responsibilities and begin work promptly.
- [ ]  Any overlap or task dependencies are resolved efficiently between teams

### Normal Flow of Events
Project manager/team leads identify the tasks to be completed during Sprint 1.
The project manager assigns tasks to appropriate teams (e.g., frontend, backend, hardware).
Each team lead reviews the tasks and provides feedback or adjustments.
Teams are notified of their tasks via GitHub Projects or a similar task-tracking tool.
Teams begin working on their assigned tasks..

### S-1:  Team faces a task they are not equipped to handle
The team encounters a task outside their skillset.
Team lead discusses with the project manager and reassigns the task to a more suitable team.

### S-2:  Overlap or dependencies between tasks
Two teams identify overlapping work or task dependencies.
Teams collaborate to ensure tasks are completed in the correct order, or dependencies are resolved.

**Comment by ktageja** [comment]
Issue closed. The task distribution has been successfully completed, with tasks assigned to the appropriate teams (frontend, backend, hardware) based on the requirements of Sprint 1. Task dependencies and overlaps were addressed, and all teams have been notified through GitHub Projects to begin their work. We are confident that this approach will ensure smooth progress for the sprint.
